_data/meltano/utilities/dagster/dagster.yml

```diff
@@ -0,0 +1,44 @@
+name: dagster
+label: Dagster
+domain_url: https://dagster.io/
+namespace: dagster
+pip_url: dagster-ext
+maintenance_status: development
+repo: https://github.com/quantile-development/dagster-ext
+executable: dagster_invoker
+logo_url: /assets/logos/utilities/dagster.jpg
+variant: quantile-development
```

Comment:
@JulesHuisman can you add an entry in https://github.com/meltano/hub/blob/main/_data/default_variants.yml#L2 with `quantile-development` as the default variant? That will enable the Hub preview build to complete.